rotkehlchen/assets/asset.py

```diff
@@ -94,6 +96,7 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     'COIN': 'COIN',
     # SLP is smooth love potion
     strethaddress_to_identifier('0xCC8Fa225D80b9c7D42F96e9570156c65D6cAAa25'): 'SLP',
+    strethaddress_to_identifier('0xBB0E17EF65F82Ab018d8EDd776e8DD940327B28b'): 'AXS',
```

Comment:
Shouldn't you also add MER here? https://github.com/rotki/rotki/issues/3361

Reply:
Sorry, was adding some missing mappings and didn't notice this mistake. Pushed again